### ISSUE TITLE
[issue]: linux i386 gui creates fat32 on first try?

### ISSUE THREAD

**Issue by Randrianasulu** [ventoy/Ventoy#1927]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.80

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

No. I didn't try these alternative boot modes.

### BIOS Mode

Both

### Partition Style

MBR

### Disk Capacity

32gb

### Disk Manufacturer

kingston

### Image file checksum (if applicable)

No.

### What happened?

I downloaded and extracted Linux version of Ventoy

-rwxr-xr-x 1 guest users 30548 Sep 21 13:43 VentoyGUI.i386*

Because I use i586 userland on x86_64 kernel I got error in log about incompatible architecture but then I just used setarch i686 and GUI showed up.

I selected my usb stick,  then hit reload button just to see what it does (it set dropdown menu back to my first big usb hdd). Selected my usb stick again.

I tried install, it warned me twice about data loss, I said sure (ok)  both times, dialog appeared after some progressbar saying everything was Ok.

I tried to copy 4.3 GB file to single partition and .... it errored out?! I blamed my selfcompiled TDE first, then noted disk was in fat32 and no obvious way to select FS from gui? I tried to create second empty partition on drive and after usual double-warning it created uefi structure ...well, at least in this case first partition was exfat, I copied my two iso files there and checked them via qemu-system-x86_64 - both booted.

I am a bit reluctant to retest on this usb stick because I'll need it tomorrow.

May be I did stupid user error. But little more info about current choice of filesystem  will be nice to have.

**Comment by ventoy** [comment]
Ventoy has  no option to select FS when install. 
It always create exfat FS for the 1st partition (ISO partition) and FAT32 for the 2nd partition (32MB VTOYEFI partition).
Did you copy the ISO file to the right partition? (you should copy to the ISO partition  not the VTOYEFI partition).
If you can reproduce the issue, please upload the log.txt and take a photo about the disk partition layout.

**Comment by Randrianasulu** [comment]
I think initially I saw just one partition, to where I copied 2 gb uso without problem. Only after I discovered it was fat32 (may be leftiver from official original fs on stick?) I re-run Ventoy with option to reserve some space at the end and got Ventoy
 and VTOYUEFI one ....

I'll try to re-test as fast as I finish planned use for this stick.

**Comment by Randrianasulu** [comment]
@
[ventoy.log](https://github.com/ventoy/Ventoy/files/9843263/ventoy.log)

strange, it says formatting exfat even at first attempt. So I guess I just got old partition map somehow?
